src/cpp/utils/dump/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+# SPDX-License-Identifier: MIT
+# Copyright (C) 2025 Advanced Micro Devices, Inc. All rights reserved.
+
+add_library(aiebu_dump_objects OBJECT
+  dump.cpp
+  ${AIEBU_SOURCE_DIR}/src/cpp/utils/target/target.cpp
+  )
+
+target_include_directories(aiebu_dump_objects
+  PRIVATE
+  ${AIEBU_SOURCE_DIR}/src/cpp
```

Comment:
This is probably a copy of some other CMakeLists.txt. 
You are also missing an include here:
```
  ${AIEBU_SOURCE_DIR}/src/cpp
  ${AIEBU_SOURCE_DIR}/src/cpp/include
  ${Boost_INCLUDE_DIRS}
```

Reply:
Updated